common/app/assets/assets.scala

```diff
@@ -152,6 +152,7 @@ class Assets(base: String, assetMapPath: String = "assets/assets.map") extends L
 
      val curl: String = RelativePathEscaper.escapeLeadingDotPaths(inlineJs("assets/curl-domReady.js"))
      val omnitureJs: String = inlineJs("assets/vendor/omniture.js")
+     val reactPath: String = "https://cdnjs.cloudflare.com/ajax/libs/react/0.13.2/react.min.js"
```

Comment:
I don't think we should bundle it, but we should host it ourselves, for when that CDN goes down. This isn't a hypothetical suggestion, it's saving us from fixing an issue that has happened in the past. I don't think, without real numbers, we can claim to be hitting the http cache well using a public CDN. There are n versions of that library, there are x CDNs, there is finite cache space, there is y percentage of other sites using react.